src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -219,7 +271,7 @@ private function createAutowiredDefinition(\ReflectionClass $typeHint, $id)
             $classOrInterface = $typeHint->isInterface() ? 'interface' : 'class';
             $matchingServices = implode(', ', $this->ambiguousServiceTypes[$typeHint->name]);
 
-            throw new RuntimeException(sprintf('Unable to autowire argument of type "%s" for the service "%s". Multiple services exist for this %s (%s).', $typeHint->name, $id, $classOrInterface, $matchingServices));
+            throw new RuntimeException(sprintf('Unable to autowire argument of type "%s" for the service "%s". Multiple services exist for this %s (%s).', $typeHint->name, $id, $classOrInterface, $matchingServices), 1);
```

Comment:
Is there any other precedent for this in the code? I can see why you used `code` - it's dead simple... and this is basically a "private" exception call: as it's thrown from a private function and is *always* caught. Let's see if anyone else thinks it should be changed.